charter/index.html

```diff
@@ -412,19 +414,19 @@ <h2>Coordination</h2>
           <h3 id="w3c-coordination">W3C Groups</h3>
           <dl>
             <dt><a href="https://www.w3.org/groups/cg/credentials">Credentials Community Group</a></dt>
-            <dd>To explore and report on the application of creating, storing, presenting, verifying, and user control of credentials in the context of Solid Protocol.</dd>
+            <dd>To explore and report on the application of creating, storing, presenting, verifying, and user control of credentials in the context of PUMPKIN Protocol.</dd>
 
             <dt><a href="https://www.w3.org/groups/wg/did">Decentralized Identifier Working Group</a></dt>
-            <dd>To investigate the areas in which existing DIDs can be adopted by the Solid Protocol (and related technical reports), as well as to advance an understanding of whether a Solid-specific DID method is needed.</dd>
+            <dd>To investigate the areas in which existing DIDs can be adopted by the PUMPKIN Protocol (and related technical reports), as well as to advance an understanding of whether a specific DID method is needed.</dd>
```

Comment:
> I think this should be out of scope

DIDs and WebID were developed largely independently, by different sub-communities, but they do overlap. And DID *is* a W3C recommendation, unlike WebID. So I don't think that the new WG can afford to simply ignore the DID WG.

Note that this does not says that the to-be-created REC will be using DIDs -- it just says that the two WG need to discuss it. The PUMPKIN WG may well come to the conclusion that, despite the overlap, DIDs are not appropriate and WebID is a better fit.

> DID is not one specification, but rather, over 100.

You could argue the same about URIs (https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml). Do you want to argue against the use of URIs as well?

>  Could you tell me if I am allowed to object

The design of the charter is not covered by the W3C process, but once the team proposes the charter to the AC, you may raise a formal objection: https://www.w3.org/2023/Process-20231103/#registering-objections

Reply:
How about:

```suggestion
            <dd>To investigate if existing DID methods should be adopted by the PUMPKIN Protocol (and related technical reports), as well as to advance an understanding of whether a specific DID method is needed.</dd>
```

(I don't see any pre-existing FOs that ask for this change, but it's always nice to improve consensus if possible.)

Reply:
```suggestion
            <dd>To investigate whether existing DID methods should be adopted by the PUMPKIN Protocol (and related technical reports), and to advance an understanding of whether a specific DID method is needed.</dd>
```

Reply:
> You could argue the same about URIs (https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml). Do you want to argue against the use of URIs as well?

No I dont want to argue against URIs.  But I do want to argue against the heavy complexity involved with DID plus the 100+ sub specs.  It's adding a very large moving part that presents.  It puts Solid at risk, either needlessly, or for political reasons.  Nothing in solid right now requires DID.

> The design of the charter is not covered by the W3C process, but once the team proposes the charter to the AC, you may raise a formal objection: https://www.w3.org/2023/Process-20231103/#registering-objections

Consider this the first objection.  IMHO the spec would be much better without this extra moving part, that is already controversial, and opens a large can of worms.  This dependency could be done elsewhere, in the CG, or later after Solid is a REC.  IMHO that is the best chance of getting solid accross the line, rather than courting objection at step one.  But this all needs a longer write-up, if the addition of this item remains a hard dependency (and I understand it may be for political reasons) we can argue the case, hopefully in a clearer way that was argued with the original DID formal objections.

Strong -1

Reply:
> The PUMPKIN WG may well come to the conclusion that, despite the overlap, DIDs are not appropriate and WebID is a better fit.

This is a huge leap.  The two identity systems are not on equal footing in Solid.  Identity is at the heart of solid.

Reply:
>  IMHO the spec would be much better without this extra moving part

The charter does not mandate those part to be in the spec; it merely states that the WG should consider it. I understand that you have a strong preference on the outcome of this discussion, but I don't think it is appropriate to prevent the WG from having it.